### ISSUE TITLE
Can panel-mipi-dbi work with a RAio RA8889 TFT LCD Controller chip based display?

### ISSUE THREAD

**Issue by Bootpanic** [notro/panel-mipi-dbi#7]
Hi,

Would it be possible to get a RAio RA8889 based display working with the panel-mipi-dbi driver?  It is about the display that is in an EdgeLogix-RPI-1000 industrial controller. A friend and I have been trying to get the display working for weeks. The EdgeLogix-RPI-1000 is built around a Raspberry Pi compute module 4. The display is connected to the CM4 via SPI. The CM4 runs Raspberry Pi OS (Debian 11) with a 6.1.61 kernel. In the meantime we have tried with panel-mipi-dbi, fbtft and anyspi without success. I've added a link to a recent kernel log and a link to an English datasheet for the RA8889 at the bottom.

Any help is welcome. Thank you in advance.

**dmesg log:**
[https://pastebin.com/kbk5YZVm](url)
**RA8889 datasheet**
[https://www.raio.com.tw/Data_raio/Datasheet/RA88%20Series/Completed/RA8889_DS_V13_Eng.pdf](url)

**Comment by notro** [comment]
I don't think that controller is MIPI DBI compatible. From the datasheet it seems to be a register based interface compared to the command based on MIPI DBI. See [wiki](https://github.com/notro/panel-mipi-dbi/wiki) for a list of commands that can tell if a controller is MIPI compatible. At a minimum 0x2a, 0x2b and 0x2c is needed for uploading the framebuffer.
